R/facet-grid-.r

```diff
@@ -50,10 +50,25 @@ NULL
 #' @param drop If `TRUE`, the default, all factor levels not used in the
 #'   data will automatically be dropped. If `FALSE`, all factor levels
 #'   will be shown, regardless of whether or not they appear in the data.
+#' @param margins either a logical value or a character
+#'   vector. Margins are additional facets which contain all the data
+#'   for each of the possible values of the faceting variables. If
+#'   `FALSE`, no additional facets are included (the
+#'   default). If `TRUE`, margins are included for all faceting
+#'   variables. If specified as a character vector, it is the names of
+#'   variables for which margins are to be created.
+#' @param facets This argument is soft-deprecated, please us `rows`
+#'   and `cols` instead.
 #' @export
 #' @examples
 #' p <- ggplot(mpg, aes(displ, cty)) + geom_point()
 #'
+#' # Use vars() to supply variables from the dataset:
+#' p + facet_grid(vars(drv))
+#' p + facet_grid(cols = vars(cyl))
+#' p + facet_grid(vars(drv), vars(cyl))
+#'
+#' # The historical formula interface is also available:
```

Comment:
Maybe classic instead of historical? I think that has a more positive connotation